src/client/components/Checker.tsx

```diff
@@ -22,24 +53,57 @@ export const Checker: FC<CheckerProps> = ({ config }) => {
     }
   }
 
+  const evaluateTransformation = (transformation: Transformation) => {
+    const { id, func, targets, args } = transformation
+    const targetValues = obtainTargetValues(targets)
+
+    switch (func) {
+      case 'NUMERICEQUAL':
+        return (transformationResults[id] = TNumericEqual(targetValues, args))
+      case 'RADIOEQUAL':
+        return (transformationResults[id] = TRadioEqual(targetValues, args))
+      case 'CHECKBOXEQUAL':
+        return (transformationResults[id] = TCheckboxEqual(targetValues, args))
+      case 'BOOLEANEQUAL':
+        return (transformationResults[id] = TBooleanEqual(targetValues, args))
+      case 'AND':
+        return (transformationResults[id] = TAnd(targetValues))
+      case 'IFTHEN':
+        if (args[0] === 'DISPLAYTEXT') {
+          if (targetValues[0]) {
+            setOutputString(args[1])
+          } else {
+            setOutputString(args[2])
+          }
+          return (transformationResults[id] = targetValues[0])
+        }
```

Comment:
the output for IFTHEN should be the strings specified as arguments depending on the expression, ie,
```suggestion
          return (transformationResults[id] = targetValues[0] ? args[1], args[2])
```

the creator then marks whichever transformation result should be displayed

Reply:
Makes sense, change committed